docs/supported_backups.md

```diff
@@ -0,0 +1,29 @@
+# OADP Supported Backups
+
+OADP supports backup and restore disaster recovery operations for both containerized and OpenShift Virtualization workloads. It is important to distinguish the nuances and support for each workload type.
+
+## Containerized Workloads
+
+OADP supports backup and restore operations for containerized workloads. 
+
+| VolumeMode | FSB - restic | FSB - kopia | CSI | CSI DataMover ** |
```

Comment:
Maybe also add another column of cloud provider snapshots ? (via VSL )

Reply:
ya.. I will before we merge. I just want to keep this clear atm for the doc team.
/hold

Reply:
@shubham-pampattiwar We'll need another dimension in the chart for that. Based on VolumeMode, CSI and native snapshots are supported in the same cases (i.e. all). We'll need to add columns for "Volume uses CSI driver" and "Cloud Volume (aws/gcp/azure)" -- so if the first new column is false, then CSI/DM are `N`, and if the second new column is false, then native snapshots are `N`

Reply:
added a section for volume snapshots